src/main/java/seedu/address/model/person/Person.java

```diff
@@ -55,10 +57,27 @@ public Email getEmail() {
     public Address getAddress() {
         return address;
     }
+
     public Note getNote() {
         return note;
     }
 
+    public Pin getPin() {
+        return this.pin;
+    }
+
+    public void toPin() {
+        pin.setPin();
+    }
+
+    public void toUnpin() {
+        pin.setUnpin();
+    }
+
```

Comment:
toPin and toUnpin sounds a bit weird to me, is pin() and unpin() enough to bring out the intution that these command are actions words?